UEBPrep/app/controllers/sessions_controller.rb

```diff
@@ -4,17 +4,23 @@ def new
   end
 
   def create
-    user = User.find_by(email: params[:session][:email].downcase, password: params[:session][:password])
-    if user
-      log_in user
-      redirect_to user
+    users = User.where(email: params[:session][:email])
+    #If it's greater than one, we violated the unique email constraint
+    #on the user model
+    if users.count == 1
+      if users.first.valid_password?(params[:session][:password])
```

Comment:
Can't check the password directly as now we only store(for security reasons) the user's password in our database in an encrypted manner.